azkaban-common/src/test/java/azkaban/project/JdbcProjectLoaderTest.java

```diff
@@ -214,6 +213,96 @@ private static void clearDB() {
     DbUtils.closeQuietly(connection);
   }
 
+    /** Test case to validated permissions for fetchProjectByName **/
+    @Test
+    public void testPermissionRetrivalByFetchProjectByName()
+        throws ProjectManagerException {
+        if (!isTestSetup()) {
+            return;
```

Comment:
just curious, if the test is not setup, should we setting up the test, or should we be returning success with the test?